crypto/asn1/a_strex.c

```diff
@@ -281,12 +281,22 @@ static int do_dump(unsigned long lflags, char_io *io_ch, void *arg,
 static const signed char tag2nbyte[] = {
     -1, -1, -1, -1, -1,         /* 0-4 */
     -1, -1, -1, -1, -1,         /* 5-9 */
-    -1, -1, 0, -1,              /* 10-13 */
-    -1, -1, -1, -1,             /* 15-17 */
```

Comment:
Note that the index `14` was missing in the original comments. It should have been `/* 14-17 */` instead.